<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telefonbuch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>761</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_search">
     <attribute name="title">
      <string>Suchen</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Vorname</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nachname</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search_firstname">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search_secondname">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>30</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Suchen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Eintr채ge</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_search_entrys">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>711</width>
        <height>211</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Vorname</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nachname</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rufnummer</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Notiz</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_add">
     <attribute name="title">
      <string>Hinzuf체gen</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_add_firstname">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_add_secondname">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nachname</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Vorname</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>30</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Hinzuf체gen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rufnummer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_add_number">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_add_note">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Notiz</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_all_entrys">
     <attribute name="title">
      <string>Alle Eintr채ge</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_all_entrys">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>711</width>
        <height>291</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Vorname</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nachname</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rufnummer</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Notiz</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar_all_entrys">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>50</y>
        <width>20</width>
        <height>251</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Teflon Buch</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
